docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -1,28 +1,30 @@
 +++
-title = "Using Stackdriver in Grafana"
-description = "Guide for using Stackdriver in Grafana"
-keywords = ["grafana", "stackdriver", "google", "guide"]
+title = "Using Google Cloud Monitoring in Grafana"
+description = "Guide for using Google Cloud Monitoring in Grafana"
+keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
 type = "docs"
-aliases = ["/docs/grafana/latest/datasources/stackdriver"]
+aliases = ["/docs/grafana/latest/datasources/stackdriver", "/docs/grafana/latest/features/datasources/stackdriver/"]
 [menu.docs]
-name = "Google Stackdriver"
+name = "Google Cloud Monitoring"
 parent = "datasources"
 weight = 4
 +++
 
-# Using Google Stackdriver in Grafana
+# Using Google Cloud Monitoring in Grafana
 
 > Available as a beta feature in Grafana v5.3.x and v5.4.x.
 > Officially released in Grafana v6.0.0
```

Comment:
```suggestion
```

Reply:
I'm not sure about this; I think this information is valuable, no?

Reply:
Not the beta info I think. How about:
```suggestion
> Officially released in Grafana v6.0.0
```

Reply:
sounds good!